Index: Dockerfile
===================================================================
diff --git a/Dockerfile b/Dockerfile
deleted file mode 100644
--- a/Dockerfile	(revision 9484474cd545e0404583ba84bdcf3f46526d7bd1)
+++ /dev/null	(revision 9484474cd545e0404583ba84bdcf3f46526d7bd1)
@@ -1,22 +0,0 @@
-# Use the official Maven image for building
-FROM maven:3.9.5 AS build
-
-# Set the working directory for building
-WORKDIR /app
-
-# Copy the application code
-COPY . .
-
-# Build the application
-RUN mvn clean package -DskipTests
-
-# Use the same Maven image for runtime to run exec:java
-FROM maven:3.9.5
-
-# Set the working directory for runtime
-WORKDIR /app
-
-COPY . .
-
-# Command to run the application
-CMD ["mvn", "exec:java", "-Dexec.mainClass=com.telefonica.cose.provenance.example.Signer", "-Dexec.args='ietf-interfaces.xml'"]
\ No newline at end of file
